src/data/prod.config.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Do we want to use TS config files instead? I like that model since it gives us the property checking.

Reply:
True will do